src/org/camunda/latera/bss/testting/TestExecution.groovy

```diff
@@ -0,0 +1,30 @@
+package org.camunda.latera.bss.testing;
+
+import org.camunda.bpm.engine.impl.pvm.runtime.ExecutionImpl
+import org.camunda.bpm.engine.runtime.Incident
+import org.yaml.snakeyaml.Yaml
+
+class TestExecution extends ExecutionImpl {
```

Comment:
Actually you don't need to extend some real class, you can just implement DelegateExecution interface.

Reply:
DelegateExecution is interface.
https://github.com/camunda/camunda-bpm-platform/blob/master/engine/src/main/java/org/camunda/bpm/engine/delegate/DelegateExecution.java#L27
If i will implement it, ill need to implement all the methods.
So i prefer to use class.